registrations/tests/test_signup_group_put.py

```diff
@@ -140,46 +184,71 @@ def test_non_created_regular_user_cannot_update_signup_group(
     default_org.regular_users.add(user)
     default_org.admin_users.remove(user)
 
-    new_extra_info = "Edited extra info"
-
     db_signup_group = SignUpGroup.objects.get(pk=signup_group.id)
-    assert db_signup_group.extra_info != new_extra_info
+    assert db_signup_group.extra_info != new_signup_group_extra_info
     assert db_signup_group.last_modified_by_id is None
 
     signup_group_data = {
         "registration": registration.id,
-        "extra_info": new_extra_info,
+        "extra_info": new_signup_group_extra_info,
     }
 
     response = update_signup_group(user_api_client, signup_group.id, signup_group_data)
     assert response.status_code == status.HTTP_403_FORBIDDEN
 
     db_signup_group.refresh_from_db()
-    assert db_signup_group.extra_info != new_extra_info
+    assert db_signup_group.extra_info != new_signup_group_extra_info
     assert db_signup_group.last_modified_by_id is None
 
 
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
-def test_admin_cannot_update_signup_group(user_api_client, registration, user):
-    signup_group = SignUpGroupFactory(registration=registration)
+def test_created_user_without_organization_can_update_signup_group(
+    organization, registration, user, user_api_client
+):
+    user.get_default_organization().admin_users.remove(user)
+
+    signup_group = SignUpGroupFactory(registration=registration, created_by=user)
+
+    db_signup_group = SignUpGroup.objects.get(pk=signup_group.id)
```

Comment:
Here as well just the factory instance could be used.

Reply:
Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/719/commits/4e2de2a39a5ba84b4dd8b8eb1375dc453555fb3b